minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -134,16 +117,15 @@ public NetAddress getWorkerAddress() {
 
   public TachyonConf newTestConf() throws IOException {
```

Comment:
I think an important goal of abstracting out a base class is to reuse settings in this method, it would be great to add `newTestConf` mechanism to `LocalTachyonClusterMultiMaster` after abstracting out the base class.